RustBoy/RustBoy.swift

```diff
@@ -148,6 +153,31 @@ fileprivate final class CoreRustBoy {
     }
 }
 
+private final class AudioHandle {
+
+    fileprivate static let sampleRate: UInt32 = 44100
```

Comment:
I feel this should come from inside an Apple library somewhere (the output sink perhaps?)

Reply:
You can push your changes for this

Reply:
Done